### ISSUE TITLE
reference npm plugins

### ISSUE THREAD

**Issue by robomatic** [juliancwirko/meteor-s-grid#5]
followed your instructions to add an npm package to the app and get this error.

 13| @import 'nib'
   ---------------^

   failed to locate @import file nib.styl

   While loading plugin `compileStylus` from package `juliancwirko:s-grid`:
   error: Can't load npm module 'nib' from packages/compileStylus_plugin.js. Check your Npm.depends().

**Comment by juliancwirko** [comment]
Hi, do you have your custom package with nib in packages folder and added to your Meteor app by `meteor add your:package-with-nib` ?

**Comment by robomatic** [comment]
yes and I tried a meteor reset followed by two meteor calls to restart.

**Comment by juliancwirko** [comment]
and your `sgrid.json` file is in place too, right?

**Comment by robomatic** [comment]
sgrid.json is in the root of myapp. myapp adds the package my:package-with-nib with a package.js file with the code:

```
Npm.depends({
  'nib': '1.1.0',
  'axis': '0.4.1'
});
```

i am also using in the same package.js file the code:

```
Package.onUse(function(api) {
  api.use ....
```

Could that be conflicting meteor?

**Comment by juliancwirko** [comment]
I don't think so. There must be some other bug :/ I will try to check this.

**Comment by juliancwirko** [comment]
hmmm I think that there is a misconception :/ I didn't saw it before. It will work if you add your npm plugins globally like `npm install -g axis nib` etc. I will try to test it more and provide some solution.

**Comment by robomatic** [comment]
Thanks for looking into this... I get the same error if they are loaded globally.

**Comment by juliancwirko** [comment]
Hey, there is a fix (v1.1.0) related to npm paths errors and Npm.depends() but I think there is also a problem with Axis itself - compilation errors. I don't know exactly what is wrong with it :/ All other plugins like `nib` or `typographic` works well. Here you will find updated documentation: https://github.com/juliancwirko/meteor-s-grid#stylus-compile-plugin-configuration
